platform-server-kernel/src/main/java/uno/cod/platform/server/core/repository/ChallengeRepository.java

```diff
@@ -37,12 +38,20 @@ Challenge findOneByTemplateAndStartDateAndOrganization(@Param("templateId") UUID
     @Query("SELECT challenge FROM Challenge challenge " +
             "LEFT JOIN FETCH challenge.challengeTemplate challengeTemplate " +
             "LEFT JOIN FETCH challengeTemplate.organization organization " +
-            "WHERE challenge.inviteOnly= :inviteOnly AND challenge.endDate > :endDate")
-    List<Challenge> findAllByInviteOnlyAndEndDateAfter(@Param("inviteOnly") Boolean inviteOnly, @Param("endDate")ZonedDateTime endDate);
+            "WHERE challenge.inviteOnly = :inviteOnly AND challenge.endDate > :endDate " +
+            "AND challenge.id NOT IN(" +
+            "SELECT challenge2.id FROM Challenge challenge2 " +
```

Comment:
it`s a hibernate fault i cannot join and search for `userId != user.id` or  `userId <> user.id` or `userId NOT IN (user.id)
`